### ISSUE TITLE
"Exit editor" button is located outside focus area

### ISSUE THREAD

**Issue by pbb72** [mapillary/mapillary_issues#2945]
When opening the sequence editor on the website, you are presented with an overlay window with lots of editing options.

There is no Cancel/Close/Exit/whatever button there however.

Only after spending a long time studying the complete screen, it appears there is an exit button in the far top right of the screen, far away from the overlay window.

This button is so far away from the focus area that it is almost impossible to discover.

![image](https://user-images.githubusercontent.com/1436366/42134576-c81c767c-7d3e-11e8-866f-a55d0fc482d8.png)

**Comment by angelday** [comment]
This is according to the original design intent and wontfix at the moment.

Most probably will be changed when we redo the UI.
